community/cypher/cypher/src/main/java/org/neo4j/cypher/internal/codegen/CompiledConversionUtils.java

```diff
@@ -395,6 +444,10 @@ public static LongStream toLongStream( Object list )
         {
             return LongStream.empty();
         }
+        else if ( list instanceof SequenceValue )
```

Comment:
can we get here?

Reply:
I don't think so (at least not in the current state of the code, it could have been the case at an earlier point during the development of this PR). We only get to this method from parameter loading, and now that parameters always comes as AnyValue and we do not convert them I don't think this method is even being called anymore.